docs/entity/features.md

```diff
@@ -65,6 +65,34 @@ const Order =
     ...
 })
 ```
+It is possible to access the fields metadata of an entity through the `schema.fields` static property:
```

Comment:
It seems to be duplicated. Why we have this content and again on the `## Metadata` session?

Lets keep only on `## Metadata`